### ISSUE TITLE
Release of Training Data

### ISSUE THREAD

**Issue by SuperN1ck** [homangab/Track-2-Act#2]
Hey!

I really like this work!
I was wondering if and in case yes, when you plan to release the processed training data or the script to generate it to retrain `Track2Act`.

Looking forward to your answer!
Cheers,
-Nick

**Comment by homangab** [comment]
Hi Nick, thanks for the question and the kind words. The training data was basically clips from the datasets mentioned in the paper and we ran Co-Tracker on these clips by specifying a grid of 400 points in the initial frame. This gives us per-timestep locations (aka tracks) of all the points in subsequent frames. For this I simply modified this script https://github.com/facebookresearch/co-tracker/blob/main/demo.py to add some mild parallelization across videos. Hope this is helpful.
